test/test_core.py

```diff
@@ -4273,6 +4275,7 @@ def do_basic_dylink_test(self, **kwargs):
     ''', 'other says 11.', 'int sidey();', force_c=True, **kwargs)
 
   @needs_dylink
+  @crossplatform_test
```

Comment:
Maybe just call this `@crossplatform` the `_test` suffix seems a little redundant.